tor/core/inbox.py

```diff
@@ -66,7 +71,12 @@ def process_reply(reply: Comment, cfg: Config) -> None:
         r_body = reply.body.lower()  # cache that thing
 
         if "image transcription" in r_body or is_comment_transcription(reply, cfg):
-            message = i18n["responses"]["general"]["transcript_on_tor_post"]
+            post_link = submission.url
+            sub_name = extract_sub_from_url(submission.url)
```

Comment:
```suggestion
            sub_name = extract_sub_from_url(post_link)
```

Reply:
Ohh yeah right!! Let's see if it works now